docs/source/configuration/environmentvariables.md

```diff
@@ -9,193 +9,186 @@ myst:
 
 # Environment variables
 
-All the environment variables defined at runtime that have the "RAZZLE_" prefix, are available in the browser under window.env
+This page describes environment variables and their usage for configuration of your Volto application at runtime.
```

Comment:
We are loosing information in this rewrite, and I think it's relevant.

Reply:
@sneridagh I think it's still here, just reordered

Reply:
@sneridagh which information is lost? I didn't really delete anything, but I shuffled things around and added some clarification.

Reply:
@sneridagh It is there: https://github.com/plone/volto/pull/5736/files#diff-00609ed769cd40cf3bc3d6fcc4431b714cb37c73cedaaea18fe9fc4c1c589597R183

Reply:
@sneridagh it was moved to the end. See https://github.com/plone/volto/pull/5736#discussion_r1480873445 and either https://github.com/plone/volto/pull/5736/files#diff-00609ed769cd40cf3bc3d6fcc4431b714cb37c73cedaaea18fe9fc4c1c589597R181 or https://deploy-preview-5736--volto.netlify.app/configuration/environmentvariables.html#access-environment-variables-in-a-browser. I find the Netlify preview easier to parse than this diff.

In any case, if you think it should be moved elsewhere, please specify where, and I can make it happen. Thank you!

Reply:
wtf, I just did a search and I overlooked it, sorry about the noise LGTM